corgidrp/detector.py

```diff
@@ -222,3 +222,26 @@ def make_detector_areas(detector_areas, areas=('image', 'prescan', 'prescan_reli
     for area in areas:
         detector_areas[area] = detector_area_mask(detector_areas, area=area)
     return detector_areas
+
+def get_rowreadtime_sec():
+    """
+    Get the value of readrowtime. The EMCCD is considered sensitive to the
+    effects of radiation damage and, if this becomes a problem, one of the
+    mitigation techniques would be to change the row read time to reduce the
+    impact of charge traps.
+ 
+    There’s no formal plan/timeline for this adjustment,  though it is possible
+    to change in the future should it need to.
+
+    Its default value is 223.5e-6 sec.
+
+    Args:
+        None
+
+    Returns:
+        Current value of rowreadtime in sec.
```

Comment:
Please check the docstring examples of other functions here to format the syntax of the Args and Returns variables.

Reply:
@semaphoreP: this function does not have arguments. I did not find an example in coorgidrp with no arguments. Is Args: None not acceptable? 

For the Return, it seems: " rowreadtime_sec (float): Current value of rowreadtime in sec." could be compliant with other examples.

Reply:
I now found the first comment. I'll add that argument. Good idea.

Reply:
You just need to remove the "Args:" section if there are no arguments. But yeah, let's add one instead!